.github/pull_request_template.md

```diff
@@ -44,8 +44,11 @@ Please explain which reason, if any, why this requires more than one approval.
 - [ ] I have separated logic changes from refactor changes (formatting, renames, etc.)
 - [ ] I have commented my code wherever necessary (can be 0)
 - [ ] I have added tests that prove my fix is effective or that my feature works, if necessary
+- [ ] I have added comprehensive [godoc](https://go.dev/blog/godoc) comments 
 
 #### Project management
 
 - [ ] I have assigned myself to this PR
 - [ ] I have linked the appropriate github issue
+- [ ] I have assigned this PR to the appropriate GitHub project
```

Comment:
As far as I can tell, there is only one project (named `go-nitro`). Is it useful to include this tag on the PRs?

(Possibly: future projects will involve new labels)

Reply:
There is only one project, but unfortunately we still need to add everything to it to get all of the cards to appear in the GitHub kanban thing -- unless anyone knows a more automatic way?

We've been forgetting to do this recently 🤷